lang/pt_BR.json

```diff
@@ -42,7 +42,7 @@
     "Yes": "Sim",
     "No": "Não",
     "Either": "Ambos",
-    "System": "Sistema",
+    "System": "Consoles",
```

Comment:
Also wondering if this may be best left as-is. On the back-end, we refer to them as systems because they may not be consoles at all, handheld or otherwise. System is more generic to fit this purpose.

Reply:
I conducted research with the platform users overall, and everyone requested the change to consoles. I know it might not make total sense, but it's because of how gamers generally refer to video games. The term 'console' becomes a general term we use (in fact, 90% of Brazilians end up including PCs along with consoles).